scripts/github-export.py

```diff
@@ -0,0 +1,132 @@
+"""
+poetry init -n
+poetry add requests
+poetry add pprintpp
+poetry run python
+"""
+
+import json
+import csv
+
+from typer import Argument
+from typer import run
+
+import requests
+from pprintpp import pprint as pp
+
+def main(
+        token: str = Argument(..., help='https://github.com/settings/tokens')
+):
+    session = requests.Session()
+    session.headers['Authorization'] = f'Bearer {token}'
+
+    pp(session.post('https://api.github.com/graphql', json={'query': '''\
+    query{
+        organization(login: "atviriduomenys"){
+          projectsNext(first: 10) {
+            nodes {
+              id
+              title
+            }
+          }
+        }
+    }
+    '''}).json())
+    # | {'id': 'PN_kwDOAKaBNM4AGx3p', 'title': 'Palaikymas'},
```

Comment:
Please remove debug comments. And add another argument - `project`, if `project` is not given then list all projects with id ant title, and exit.

If project id is given, then do not list projects and proceed with the export.

Reply:
Comments removed. New argument project_id added.